.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+jobs:
+  lint:
+    docker:
+    - image: python:3.6.4
+    steps:
+    - checkout
+    - run:
+        name: Lint (flake8)
+        command: |
+          pip install -r requirements-lint.txt
+          flake8 relayer tests
+  unit_test:
+    docker:
+    - image: python:3.5.5
```

Comment:
That was my fault, I started using 3.6.4 for the linters because it doesn't really matter. For unit testing, we're using 3.5.5 because that's what our base images are built on.